sys-fs/udiskie/udiskie-1.7.5.ebuild

```diff
@@ -0,0 +1,54 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+PYTHON_COMPAT=( python3_4 python3_5 python3_6 )
+inherit distutils-r1 gnome2-utils
+
+DESCRIPTION="An automatic disk mounting service using udisks"
+HOMEPAGE="https://pypi.org/project/udiskie/ https://github.com/coldfix/udiskie"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~x86"
+IUSE=""
+
+RDEPEND="dev-python/docopt[${PYTHON_USEDEP}]
+	dev-python/pygobject:3[${PYTHON_USEDEP}]
+	dev-python/pyxdg[${PYTHON_USEDEP}]
+	dev-python/pyyaml[${PYTHON_USEDEP}]
+	sys-fs/udisks:2"
+DEPEND="app-text/asciidoc
+	dev-python/setuptools[${PYTHON_USEDEP}]"
+
+src_prepare() {
+	sed -i \
+		-e "s:udiskie.__version__:2:" \
```

Comment:
Why does that do and why `2`? Please add a comment over `sed` because people are going to scratch their head over this.

Reply:
Did you find an explanation or did you just bluntly remove it without understanding why it was added and whether it is still necessary?

Reply:
While tempting to just delete the stuff, I actually had a look in the package. What it does is force the version to be reported as 2. Versioning in the package looks fine to me, so no need for this as far as I can tell.